src/FirebaseHttpClient.h

```diff
@@ -38,6 +38,6 @@ class FirebaseHttpClient {
 };
 
 static const char kFirebaseFingerprint[] =
-      "7A 54 06 9B DC 7A 25 B3 86 8D 66 53 48 2C 0B 96 42 C7 B3 0A";
+      "9A E1 A3 B7 88 E0 C9 A3 3F 13 72 4E B5 CB C7 27 41 B2 0F 6A";
```

Comment:
SSL Server Certificate
Issued To

Common Name (CN)	firebaseio.com
Organization (O)	Google Inc
Organizational Unit (OU)	<Not Part Of Certificate>
Issued By

Common Name (CN)	Google Internet Authority G2
Organization (O)	Google Inc
Organizational Unit (OU)	<Not Part Of Certificate>
Validity Period

Issued On	Monday, December 5, 2016 at 10:34:50 AM
Expires On	Sunday, August 27, 2017 at 5:00:00 PM
Fingerprints

SHA-256 Fingerprint	8C D9 B1 F9 1C FD E1 55 48 54 DB 94 DB 8A C4 17
2F 5C D7 2B 6F BF 93 2A BD 60 31 EF 85 84 2B 09
SHA-1 Fingerprint	9A E1 A3 B7 88 E0 C9 A3 3F 13 72 4E B5 CB C7 27
41 B2 0F 6A